_targets.R

```diff
@@ -1,16 +1,19 @@
 
 library(targets)
+library(tarchetypes)
+library(tibble)
 
 options(tidyverse.quiet = TRUE)
-tar_option_set(packages = c("tidyverse", "dataRetrieval", "urbnmapr", "rnaturalearth", "cowplot"))
+tar_option_set(packages = c("tidyverse", "dataRetrieval", "urbnmapr", "rnaturalearth", "cowplot",
+                            "tarchetypes", "tibble"))
```

Comment:
Use `tar_option_set` for packages required to build your targets, and the packages imported using `library()` are those needed to run the `_targets.R` script. Since `targets`, `tarchetypes`, and `tibble` are used to run the pipeline itself, keep those outside `tar_option_set`

Reply:
Got it!